docs/blog/newsletter-june-2021.rst

```diff
@@ -0,0 +1,138 @@
+
+.. post:: June 03, 2021
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – June 2021
+#####################################
+
+Hi everybody! It's Beth here, writing to you from my gloriously sunny balcony (thank *goodness* rainy May is over). I hope you're all enjoying the weather wherever you are.
+
+The big thing I want to shout about this month is the `Prague Call for Proposals </conf/prague/2021/cfp/>`__. If you were even sort-of entertaining the thought of submitting a talk idea, take this as your sign to go for it! My favourite thing about giving a conference talk is the conversations the talk starts. It's awesome for helping you find people who care about the same things as you; so if there's something you'd really like to see a talk about this year, I really encourage you to submit your idea.
+
+On to the stories from Slack! This month we've been talking about prioritization, the challenges of having busy colleagues, and much much more.
+
+-----------------------------------------------------------
+Priorities: Document new features or improve existing docs?
+-----------------------------------------------------------
+
+What would you prioritize: customer-requested improvements to existing documentation or working on new feature documentation? We discussed it on Slack this month as an interview question; for many documentarians though, the tension between priotizing new vs. existing docs is a day-to-day reality. Mostly, this means prioritizing which to work on *first* - because neither releasing new features without documentation nor neglecting to update existing documentation are good options.
```

Comment:
Change "...the tension between **priotizing** new..." to "...the tension between **prioritizing** new..."